FASTER/Views/Deployment.xaml

```diff
@@ -129,5 +133,40 @@
                 </DataGridTextColumn>
             </DataGrid.Columns>
         </DataGrid>
+
+        <Grid Grid.Row="2" Margin="50px 0px 50px 10px">
+            <Grid.ColumnDefinitions>
+                <ColumnDefinition Width="600px"/>
```

Comment:
I'm not too crazy about hard-coding these, but I'm not sure what specific sizing you'd want to go with.